hadoop-cloud/pom.xml

```diff
@@ -200,6 +200,46 @@
     -->
     <profile>
       <id>hadoop-3.2</id>
+      <properties>
+        <extra.source.dir>src/hadoop-3/main/scala</extra.source.dir>
+        <extra.testsource.dir>src/hadoop-3/test/scala</extra.testsource.dir>
+      </properties>
+
+      <build>
+        <plugins>
+          <!-- Include a source dir for Hadoop 3.2+ only -->
```

Comment:
Unnecessary comment (this is under the profile after all).

Reply:
ok